dts/arm/st/f1/stm32f103Xg.dtsi

```diff
@@ -0,0 +1,162 @@
+/* SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2020 Google, LLC
+ *
+ * SoC device tree include for STM32F103xG SoCs
+ * where 'x' is replaced for specific SoCs like {R,V,Z}
+ */
+
+#include <mem.h>
+#include <st/f1/stm32f103Xe.dtsi>
+
+/ {
+	sram0: memory@20000000 {
+		reg = <0x20000000 DT_SIZE_K(96)>;
+	};
+
+	soc {
+		flash-controller@40022000 {
+			flash0: flash@8000000 {
+				/* Note that there are actually two banks of
+				 * flash (512KB each) and two flash controllers.
+				 * This matters if you're doing in-application
+				 * flash programming and you need the
+				 * read-while-write capabilities, but is
+				 * otherwise a non-issue.
+				 */
+				reg = <0x08000000 DT_SIZE_K(1024)>;
+				erase-block-size = <DT_SIZE_K(2)>;
+			};
+		};
+
+		adc2: adc@40012800 {
+			compatible = "st,stm32-adc";
+			reg = <0x40012800 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000400>;
+			/* Shares vector with ADC1 */
+			interrupts = <18 0>;
+			status = "disabled";
+			label = "ADC_2";
+			#io-channel-cells = <1>;
+		};
+
+		adc3: adc@40013c00 {
+			compatible = "st,stm32-adc";
+			reg = <0x40013c00 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00008000>;
+			interrupts = <47 0>;
+			status = "disabled";
+			label = "ADC_3";
+			#io-channel-cells = <1>;
+		};
```

Comment:
Should be made available on `dts/arm/st/f1/stm32f103Xe.dtsi`

Reply:
Done